lib/pg_ha_migrations/struct.rb

```diff
@@ -0,0 +1,59 @@
+module PgHaMigrations::Struct
+  Table = Struct.new(:name, :schema) do
+    delegate :inspect, to: :name
+
+    def fully_qualified_name
+      @fully_qualified_name ||= [
+        ActiveRecord::Base.connection.quote_schema_name(schema),
+        ActiveRecord::Base.connection.quote_table_name(name),
+      ].join(".")
+    end
```

Comment:
We _could_ add even more methods here, like `partitioned?` and `valid?` (for the index struct below), but I like that these are data objects that only contain light validation and formatting

Reply:
If we don't have a need for them now, that's fine, but I wouldn't shy from adding them if it made the code cleaner.